### DISCUSSION TITLE
error /vercel/path0/node_modules/sharp: Command failed.

### DISCUSSION THREAD

**Discussion by coelmay** [vercel/vercel#6373] [Help] [answered]
Hi
Have reached the deploy stage of Mastering Nuxt. This is my first time deploying on **Vercel**. But unlike in the video, I didn't see **Congratulations** on deploy. Build failed starting from here
```
11:07:03.312 | [4/5] Building fresh packages...
-- | --
11:07:06.396 | error /vercel/path0/node_modules/sharp: Command failed.
11:07:06.396 | Exit code: 1
11:07:06.396 | Command: (node install/libvips &amp;&amp; node install/dll-copy &amp;&amp; prebuild-install) \|\| (node install/can-compile &amp;&amp; node-gyp rebuild &amp;&amp; node install/dll-copy)
11:07:06.396 | Arguments:
11:07:06.396 | Directory: /vercel/path0/node_modules/sharp
11:07:06.396 | Output:
11:07:06.396 | sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.6/libvips-8.10.6-linux-x64.tar.br
11:07:06.396 | /bin/sh: prebuild-install: command not found
11:07:06.396 | gyp info it worked if it ends with ok
```
followed by numerous `gyp info ...` then
```
11:07:06.398 | ../src/common.cc:24:10: fatal error: vips/vips8: No such file or directory
-- | --
11:07:06.398 | #include &lt;vips/vips8&gt;
11:07:06.398 | ^~~~~~~~~~~~
11:07:06.398 | compilation terminated.
11:07:06.398 | make: *** [Release/obj.target/sharp/src/common.o] Error 1
11:07:06.398 | make: Leaving directory `/vercel/path0/node_modules_dev/sharp/build'
11:07:06.398 | gyp ERR! build error
11:07:06.398 | gyp ERR! stack Error: `make` failed with exit code: 2
```
end ending in
```
11:07:06.435 | Error: Command "yarn install" exited with 1
```

I don't know why the `sharp` module is there because it isn't part of the project (it isn't in the `node_modules` directory in my local environment at least.)
Any clues as to the source of the error, and how to remedy it greatly appreciated.

**Comment by leerob** [comment]
Hey! I'm sorry you ran into this. Is your code public I could take a look at?

**Reply by coelmay** [reply]
Hey @leerob

It isn't, only because I figured not a good idea to have **paid for** code public (plus I never quite got to the point of setting up environment variables so API keys are still in there.) Build log URL of any use?

**Reply by coelmay** [reply]
Fixed environment variables. Tried deploying again with same issue.
Repository: ~~https://github.com/coelmay/nuxtbnb2~~

**Comment by coelmay** [comment, marked as answer]
Was something wrong with my package.json and/or yarn.lock. Deployed with success.

**Reply by ezracarr** [reply]
What was wrong with it?  I have the same issue

**Reply by derekgygax** [reply]
Please tell us what was wrong with your package.json file

**Reply by sergical** [reply]
I had a similar issue. Turns out I installed something with `npm` instead of `yarn` and for some reason `npmjs.org` was referenced in a `yarn.lock` file. Nuking `yarn.lock` and running `yarn install` again and deploying resolved the issue

**Comment by julianesp** [comment]
Soy otro con este error:
npm ERR! command sh -c -- (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
Si lo solucionan, por favor comenten.
